src/components/BottomSheet.tsx

```diff
@@ -0,0 +1,261 @@
+                        snapPoints={snapPoints}
+                        index={0}
+                        enableHandlePanningGesture={false}
+                        handleIndicatorStyle={styles.handleIndicator}
+                        onChange={handleSheetChanges}
+                    >
+                        <BottomSheetScrollView showsVerticalScrollIndicator={false}>
+                            <View style={styles.buttonContainer}>
+                                <TouchableOpacity style={styles.button} onPress={switchWeather}>
+                                    <Text
+                                        style={[styles.weather, { color: sectionWeather ? colors.BLACK : colors.GRAY }]}
+                                    >
+                                        {BottomSheetButtons.WEATHER}
+                                    </Text>
+                                </TouchableOpacity>
+                                <TouchableOpacity style={styles.button} onPress={switchСlothes}>
+                                    <Text
+                                        style={[styles.clothes, { color: sectionСlothes ? colors.BLACK : colors.GRAY }]}
+                                    >
+                                        {BottomSheetButtons.CLOTHES}
+                                    </Text>
+                                </TouchableOpacity>
+                            </View>
+                            <WeatherComponent
+                                warningType={WarningType.Thunderstorm}
+                                // todo: change to props
+                                timeRelatedArray={[
+                                    {
+                                        time: '2023-03-19 20:00:00.000',
+                                        weatherType: WeatherType.Sunny,
+                                        temperature: 12,
+                                    },
+                                    {
+                                        time: '2022-09-27 21:00:00.000',
+                                        weatherType: WeatherType.Sunny,
+                                        temperature: 12,
+                                    },
+                                    {
+                                        time: '2022-09-27 22:00:00.000',
+                                        weatherType: WeatherType.Sunny,
+                                        temperature: 12,
+                                    },
+                                    {
+                                        time: '2022-09-27 23:00:00.000',
+                                        weatherType: WeatherType.Sunny,
+                                        temperature: 12,
+                                    },
+                                    {
+                                        time: '2022-09-27 24:00:00.000',
+                                        weatherType: WeatherType.Sunny,
```

Comment:
Do not push with example props, you could create object in app.tsx and pass it is props. It is very hard to see all structure easily now